scss/application.scss

```diff
@@ -164,13 +137,50 @@ body {
       height: $triangle-height;
     }
   }
+
+  &-content {
+    width: 100%;
+    max-width: 1200px;
+    display: flex;
+    margin: 60px auto;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    display: grid;
+    align-content: space-evenly;
+    justify-content: space-evenly;
+    grid-template-columns: $tech-card-width $tech-card-width $tech-card-width;
+    grid-template-rows: $tech-card-height $tech-card-height
+  }
+
 }
 
-.dev-content {
-  width: 100%;
-  max-width: 1200px;
+.item-a {
+  align-self: center;
+  grid-column: 1;
+  grid-row: 1 / 3;
+  height: 100%;
   display: flex;
-  margin: 60px auto;
-  flex-flow: row wrap;
-  justify-content: flex-start;
+  align-items: center;
+  border-right: 2px solid $green;
+  border-radius: 10px;
+}
+
+.item-b {
+  grid-column: 2 / 3;
+  grid-row: 1 / 3;
```

Comment:
There's no need to set this properties to each element in the grid. The only one that has different positioning is the first one, so you can set only that one and the rest will know where to position themselves